go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/PrathameshMutkure/go-gci19
 
 go 1.13
 
-require github.com/gin-gonic/gin v1.5.0 // indirect
+require github.com/gin-gonic/gin v1.5.0
```

Comment:
This was an automatic change, don't know if it was part of the process, shoul I revert this change?